arithmetization/src/main/java/net/consensys/linea/zktracer/module/oob/OobChunk.java

```diff
@@ -0,0 +1,1294 @@
+                    EWord.of(mbs).loBigInt(),
+                    EWord.of(bbs).loBigInt(),
+                    true);
+          }
+          oobParameters = prcModexpXbsParameters;
+          setModexpXbs(prcModexpXbsParameters);
+        } else if (isModexpLead) {
+          final ModexpLeadParameters prcModexpLeadParameters =
+              new ModexpLeadParameters(bbs, cds, ebs);
+
+          oobParameters = prcModexpLeadParameters;
+          setModexpLead(prcModexpLeadParameters);
+        } else if (prcModexpPricing) {
+          int maxMbsBbs = max(mbs.intValue(), bbs.intValue());
+          final ModexpPricingParameters prcModexpPricingParameters =
+              new ModexpPricingParameters(callGas, returnAtCapacity, exponentLog, maxMbsBbs);
+
+          oobParameters = prcModexpPricingParameters;
+          setPrcModexpPricing(prcModexpPricingParameters);
+        } else if (prcModexpExtract) {
+          final ModexpExtractParameters prcModexpExtractParameters =
+              new ModexpExtractParameters(cds, bbs, ebs, mbs);
+
+          oobParameters = prcModexpExtractParameters;
+          setPrcModexpExtract(prcModexpExtractParameters);
+        }
+      } else if (isBlakePrecompile()) {
+        if (isBlake2FCds) {
+          final Blake2fCallDataSizeParameters prcBlake2FCdsParameters =
+              new Blake2fCallDataSizeParameters(cds, returnAtCapacity);
+
+          oobParameters = prcBlake2FCdsParameters;
+          setBlake2FCds(prcBlake2FCdsParameters);
+        } else if (isBlake2FParams) {
+          final BigInteger blakeR =
+              frame
+                  .shadowReadMemory(argsOffset, cds.longValue())
+                  .slice(0, 4)
+                  .toUnsignedBigInteger();
+
+          final BigInteger blakeF =
+              BigInteger.valueOf(
+                  toUnsignedInt(frame.shadowReadMemory(argsOffset, cds.longValue()).get(212)));
+
+          final Blake2fParamsParameters prcBlake2FParamsParameters =
+              new Blake2fParamsParameters(callGas, blakeR, blakeF);
+
+          oobParameters = prcBlake2FParamsParameters;
+          setBlake2FParams(prcBlake2FParamsParameters);
+        }
+      } else {
+        throw new RuntimeException("no opcode or precompile flag was set to true");
+      }
+    }
+  }
+
+  // Constraint systems for populating lookups
+  private void callToADD(
+      int k, BigInteger arg1Hi, BigInteger arg1Lo, BigInteger arg2Hi, BigInteger arg2Lo) {
+    final EWord arg1 = EWord.of(arg1Hi, arg1Lo);
+    final EWord arg2 = EWord.of(arg2Hi, arg2Lo);
+    addFlag[k] = true;
+    modFlag[k] = false;
+    wcpFlag[k] = false;
+    outgoingInst[k] = UnsignedByte.of(OpCode.ADD.byteValue());
+    outgoingData1[k] = arg1Hi;
+    outgoingData2[k] = arg1Lo;
+    outgoingData3[k] = arg2Hi;
+    outgoingData4[k] = arg2Lo;
+    outgoingResLo[k] =
+        BigInteger.ZERO; // This value is never used and BigInteger.ZERO is a dummy one
+
+    // lookup
+    add.callADD(arg1, arg2);
```

Comment:
you don't use the result of ADD in OOB trace ?

Reply:
This is the systematic mistake I have been done in some spot. I'll fix it. thanks!

Reply:
This was the case for the previous version of oob. I need to review it for the new one. I'll just kill that useless row for now.